spec/services/geographical_area_import_service_spec.rb

```diff
@@ -3,28 +3,28 @@
 RSpec.describe GeographicalAreasImportService do
   subject(:importer) { described_class.new }
 
-  let(:hjid_mappings_file) { Rails.root.join('spec', 'fixtures', 'files', 'hjid_to_sid_map.csv').to_s }
-  
-  let!(:area) {
+  let(:hjid_mappings_file) { Rails.root.join('spec/fixtures/files/hjid_to_sid_map.csv').to_s }
```

Comment:
Hmm is Rubocop asking you to do it like this? It's a bit unusual.

Reply:
```suggestion
  let(:hjid_mappings_file) { Rails.root.join('spec', 'fixtures', 'files', 'hjid_to_sid_map.csv').to_s }
```